docs/Time_in_Arbitrum.md

```diff
@@ -0,0 +1,48 @@
+# Time in Arbitrum Rollup
+
+As on Ethereum, Arbitrum doesn't use real wall-clock time, but instead uses block numbers, which correspond to block numbers on the main Ethereum chain.
+
+Also as on Ethereum, clients submit transactions they want to see happen, and the system (usually) executes those transactions at some later time.
+In Arbitrum Rollup, clients submit transactions by posting messages to the Ethereum chain.
+These messages are put into the chain's *pending inbox*.
+Every message is timestamped with the time it was put into the pending inbox.
+
+Messages in an ArbChain's pending inbox are processed in order.
+Generally, some time will elapse between the time when a message is put into the pending inbox (and timestamped) and the time when the contract processes the message and carries out the transaction requested by the message.
+
+If your Solidity contract, running on Arbitrum Rollup, accesses ``block.number``, this will give you the ArbChain's current time. 
+If you want to know when the current message (the one that requested the transaction you're currently running) was timestamped, you can get it in Solidity by writing:
+
+    ArbSys(address(100)).currentMessageTime()
+
+Because ArbChains operate off-chain, an ArbChain's time might differ a little bit from the current Ethereum block time.
+The ArbChain's clock will never get ahead of the Ethereum block time, but the ArbChain's clock might "run a bit slow". 
+In particular, it might be as many as 20 blocks behind the true Ethereum time.
```

Comment:
Are we settling on 20, or is this going to be parameterized and 20 is an example?

Reply:
We can change this later, but I wanted to put in a concrete value, and 20 seems reasonable.  Happy to change it if others think another number is better.

Reply:
Sounds good to me. 20 sounds good to me as a place to start.